src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1548,7 +1548,7 @@ Scheduler start can be customized by automatically triggering user's scripts.
 A list of scripts can be specified in 'PROACTIVE_HOME/config/scheduler/settings.ini`
```

Comment:
could you please put PROACTIVE_HOME/config/scheduler/settings.ini under `PROACTIVE_HOME/config/scheduler/settings.ini` (replace ' with `)

Reply:
yeap, thanks i will fix it